src/launcher/StracciatellaLauncher.cc

```diff
@@ -61,39 +61,42 @@ StracciatellaLauncher::StracciatellaLauncher() {
       { Fl_Group* o = new Fl_Group(0, 50, 520, 300, "@-> Mods ");
         o->labelcolor((Fl_Color)24);
         o->hide();
-        { Fl_Group* o = new Fl_Group(10, 60, 500, 165);
-          { enabledModsBrowser = new Fl_Browser(20, 75, 215, 150, "Enabled Mods:");
+        { Fl_Group* o = new Fl_Group(10, 80, 500, 165);
+          { enabledModsBrowser = new Fl_Browser(20, 95, 215, 150, "Enabled Mods:");
             enabledModsBrowser->type(3);
             enabledModsBrowser->align(Fl_Align(FL_ALIGN_TOP_LEFT));
           } // Fl_Browser* enabledModsBrowser
-          { enableModsButton = new Fl_Button(245, 75, 30, 30, "@<");
+          { enableModsButton = new Fl_Button(245, 95, 30, 30, "@<");
             enableModsButton->tooltip("Enable one or multiple mods.");
             enableModsButton->deactivate();
           } // Fl_Button* enableModsButton
-          { disableModsButton = new Fl_Button(245, 115, 30, 30, "@>");
+          { disableModsButton = new Fl_Button(245, 135, 30, 30, "@>");
             disableModsButton->tooltip("Disable one or multiple mods.");
             disableModsButton->deactivate();
           } // Fl_Button* disableModsButton
-          { moveUpModsButton = new Fl_Button(245, 155, 30, 30, "@2<");
-            moveUpModsButton->tooltip("Increase mod priority. If a resource file is present in multiple mods, the fi\
-le from the mod with the highest priority is used.");
+          { moveUpModsButton = new Fl_Button(245, 175, 30, 30, "@2<");
+            moveUpModsButton->tooltip("Move mod up in the list and decrease mod priority..");
```

Comment:
Here and in the next  chunk it should either be one full stop or three (or the Unicode ellipsis if anyone wants to be pedantic).